src/reflection.cpp

```diff
@@ -481,4 +481,232 @@ Offset<const Table *> CopyTable(FlatBufferBuilder &fbb,
   }
 }
 
+bool VerifyStruct(flatbuffers::Verifier &v,
+                  const flatbuffers::Table &parentTable,
```

Comment:
Please use Google C++ Style Guide for all indentifiers, so in this case `parent_table`.